Resources/Prototypes/Entities/Structures/Piping/Disposal/pipes.yml

```diff
@@ -16,6 +16,13 @@
   - type: Physics
     bodyType: Static
   - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+        bounds: "-0.25,-0.5,0.25,0.5"
+      mask:
+      - Impassable
+      - LowImpassable
```

Comment:
these subfloor objects could use an enum, its the same as `SmallMobMask` currently but that is a bit weird to give to these